core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -3377,10 +3377,16 @@ object SparkContext extends Logging {
     }
   }
 
-  private def getClusterManager(url: String): Option[ExternalClusterManager] = {
+  private[spark] def getClusterManager(url: String): Option[ExternalClusterManager] = {
     val loader = Utils.getContextOrSparkClassLoader
     val serviceLoaders =
-      ServiceLoader.load(classOf[ExternalClusterManager], loader).asScala.filter(_.canCreate(url))
+      ServiceLoader.
+        load(classOf[ExternalClusterManager], loader).
+        asScala.
+        filter(
+          a =>
+            a.canCreate(url)
+        )
```

Comment:
This code looks the same as line 3383.  is this just a whitespace change?

Reply:
Well,

    _.canCreate(url)

changed into

    a => a.canCreate(url)

which is equivalent but causes the line breaks as the line gets too long.

I'd prefer the former.

Reply:
Accidental change, will revert